MeshLib/Elements/LineRule2.h

```diff
@@ -39,6 +39,9 @@ class LineRule2 : public EdgeRule
     /// Constant: The FEM type of the element
     static const CellType cell_type = CellType::LINE2;
 
+    /// Constant: The number of edges
+    static const unsigned n_edges = 1;
```

Comment:
I don't think so, this a general issue I stumbled upon while doing this PR. You can return minimum and maximum edge lengths from any mesh. This is used in various analytical geometry algorithms. However, if the mesh in question is a mesh consisting completely of lines, the getMinEdgeLength() and getMaxEdgeLength() methods will return corrupted values (namely max-limits for min and 0 for max). This line fixes the issue and apparently also causes no problems with any of the benchmarks.